runtime/src/main/scala/akka/grpc/scaladsl/Metadata.scala

```diff
@@ -18,9 +23,45 @@ import akka.util.ByteString
    *         that is a text key is used.
    */
   def getText(key: String): Option[String]
+
   /**
    * @return The binary header value for `key` if one exists, if the same key has multiple values the last occurrence
    *         that is a text key is used.
    */
   def getBinary(key: String): Option[ByteString]
+
+  /**
+   * @return The metadata as a map.
+   */
+  def asMap: MetadataMap
+}
+
+class MetadataImpl(headers: immutable.Seq[HttpHeader] = immutable.Seq.empty) extends Metadata {
+  lazy private val b64Decoder = Base64.getDecoder
```

Comment:
This one is just returning a static final field in `java.util.Encoder` no need to introduce a lazy val for that.

Reply:
Fixed